docs/uploadAemAsset.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: AEM asset upload API
+---
+
+<p>Campaign API for downloading & uploading(publishing) AEM cloud asset on campaign publication server. Just like all APIs in the SDK, it's been wrapped into a function and will return a JSON object containing publishedUrl.</p>
+<p class="warning">Note: This API is restricted to be used from acc-web-ui only. It is used when an asset is selected from
+    AEM repository using asset-picker inside email-designer(acrite).
+    The API with different params/version will be extended to be used without acc-web-ui to download & publish AEM assets
+    in campaign in future.  
+</p>
+<h1>/nms/aemAssetDownload.jssp</h1>
+<p>This API is authenticated only using ims user token and run directly on the Apache front server.</p>
+<pre class="code">
+const version = sdk.getSDKVersion();
+console.log(`${version.description} version ${version.version}`);
+
+<span class="comment">// Logon to a Campaign instance with ImsBearerToken</span>
+const connectionParameters = sdk.ConnectionParameters.ofImsBearerToken("http://acc-sdk:8080", "ey...", options);
+console.log(connectionParameters);
+
+const client = await sdk.init(connectionParameters);
+await client.logon();
+
+<span class="comment">// This url is download url link provided by acrite from selection handlebar</span>
+<span class="comment">// when asset is selected in asset-picker UI component.</span>
+const url = "https://abcd.adobeaemcloud/asset/xyz.png";
+
+try {
+  const response = await client.fileUploader.uploadAemAsset(url);
+  console.log(response);
+}catch(e){
```

Comment:
I don't think this try/catch is useful. For the sake of the simplicity of the example, could you remove it?